### ISSUE TITLE
Update table on the new Data Dashboard page to use the new and improved Table component

### ISSUE THREAD

**Issue by penny-lischer** [CDCgov/prime-reportstream#9388]
## Problem statement

As part of the STLT Dashboard, we need to implement the design of the table on the dashboard page. This implementation is strictly visual, and we'll have a separate ticket for hooking it up to the API afterward.

## What you need to know

 ![Screenshot 2023-04-12 at 12.19.15.png](https://images.zenhubusercontent.com/5999b3b38f62dc7798c36942/fa35ad15-388d-44bc-9c56-24ee830135aa)

[Figma](https://www.figma.com/file/6mwI5ac6rprACKDzDo4Ady/ReportStream-Workspace-%7C-2023?node-id=2369-26965&t=CaF9GFXFAtXq9SzI-0)

## Acceptance criteria

- [ ] Table should use the new and improved Table component
- [ ] Table should sort on all columns
- [ ] Table should paginate
- [ ] Table should refresh data when filter button is clicked
- [ ] Clicking on ReportId will take you to the existing Report Details page /report-details/
*Do not link to Provider/Submitter/facilities within rows for now.

## To do

- [ ] Update Table to use the new and improved Table component
- [ ] Table should sort on all columns
- [ ] Table should paginate
- [ ] Table should refresh data when filter button is clicked
- [ ] Clicking on ReportId will take you to the existing Report Details page /report-details/
*Do not link to Provider/Submitter/facilities within rows for now.

**Comment by Andrey-Glazkv** [comment]
Blocked by the EoD 5/11

**Comment by chris-kuryak** [comment]
No longer needed. Table component already in. Styling will be updated by #10626.
